source/gent/tier1_hortense.rst

```diff
@@ -526,7 +543,7 @@ The ``install.sh`` script should be implemented such that it installs the softwa
 with ``/readonly/``.
 
 Or you can start a new shell session in which ``/readonly/$VSC_SCRATCH_PROJECTS_BASE/...`` is
-accessible with write permissions:
+accessible with sort-of write permissions:
```

Comment:
I would only add "sort-of" if you explain why you call it "sort-of"... That's already clarified above, no need to here imho.